potato/base_htmls/home.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<title>Login V1</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<link rel="icon" type="image/png" href="https://colorlib.com/etc/lf/Login_v1/images/icons/favicon.ico">
+
+<link rel="stylesheet" type="text/css" href="home_files/bootstrap.css">
+
+<link rel="stylesheet" type="text/css" href="home_files/font-awesome.css">
+
+<link rel="stylesheet" type="text/css" href="home_files/animate.css">
+
+<link rel="stylesheet" type="text/css" href="home_files/hamburgers.css">
+
+<link rel="stylesheet" type="text/css" href="home_files/select2.css">
+
+<link rel="stylesheet" type="text/css" href="home_files/util.css">
+<link rel="stylesheet" type="text/css" href="home_files/main.css">
+
+<meta name="robots" content="noindex, follow">
+</head>
+<body>
+<div class="limiter">
+<div class="container-login100">
+<div class="wrap-login100">
+<div class="login100-pic js-tilt" data-tilt="" style="will-change: transform; transform: perspective(300px) rotateX(0deg) rotateY(0deg);">
+<img src="home_files/img-01.webp" alt="IMG">
+</div>
+<form class="login100-form validate-form">
+<span class="login100-form-title">
+Member Login
+</span>
+<div class="wrap-input100 validate-input" data-validate="Valid email is required: ex@abc.xyz">
+<input class="input100" type="text" name="email" placeholder="Email">
+<span class="focus-input100"></span>
+<span class="symbol-input100">
+<i class="fa fa-envelope" aria-hidden="true"></i>
+</span>
+</div>
+<div class="wrap-input100 validate-input" data-validate="Password is required">
+<input class="input100" type="password" name="pass" placeholder="Password">
+<span class="focus-input100"></span>
+<span class="symbol-input100">
+<i class="fa fa-lock" aria-hidden="true"></i>
+</span>
+</div>
+<div class="container-login100-form-btn">
+<button class="login100-form-btn">
+Login
+</button>
+</div>
+<div class="text-center p-t-12">
+<span class="txt1">
+Forgot
+</span>
+<a class="txt2" href="#">
+Username / Password?
+</a>
+</div>
+<div class="text-center p-t-136">
+<a class="txt2" href="#">
+Create your Account
+<i class="fa fa-long-arrow-right m-l-5" aria-hidden="true"></i>
+</a>
+</div>
+</form>
+</div>
+</div>
+</div>
+
+<script src="home_files/jquery-3.js"></script>
+
+<script src="home_files/popper.js"></script>
+<script src="home_files/bootstrap.js"></script>
+
+<script src="home_files/select2.js"></script>
+
+<script src="home_files/tilt.js"></script>
+<script>
+		$('.js-tilt').tilt({
+			scale: 1.1
+		})
+	</script>
+
+<script async="" src="home_files/js"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'UA-23581568-13');
```

Comment:
Where is this google analytics tracker going to?

Reply:
this probably came from some really old code. This file was not created with a git mv so everything is showing as difference here. I can clean this a bit

Reply:
fixed with [9f7d6f6](https://github.com/davidjurgens/potato/pull/32/commits/9f7d6f61dbcefe8fcf541ff33051e5621aaa84f4)